examples/api-basic/api/v1/app.go

```diff
@@ -0,0 +1,43 @@
+package v1
+
+import (
+	"reflect"
+
+	"github.com/caicloud/nirvana/definition"
+	"github.com/caicloud/nirvana/examples/api-basic/api/v1/operators"
+	"github.com/caicloud/nirvana/examples/api-basic/application"
+)
+
+func init() {
+	register(app)
+}
+
+var app = definition.Descriptor{
+	Path: "/applications",
+	Definitions: []definition.Definition{
+		{
+			Method:   definition.Create,
+			Function: application.CreateApplication,
+			Parameters: []definition.Parameter{
+				{
+					Source: definition.Body,
+					Type:   reflect.TypeOf(&application.ApplicationV1{}),
```

Comment:
I'd rather write another `Function` to handle the convert process.

Reply:
You think 
```go
func AConverter(ctx context.Context, filed string, xxx sometype, obj *A) (*A,error)
func BConverter(ctx context.Context, filedxx int, obj *B) (*B,error)
```
is better than
```go
func AConverter(ctx context.Context, obj *A)  (*A,error)
func BConverter(ctx context.Context, obj *B)  (*B,error)
```
?

Reply:
Yes!!!!!